source/clear-linux/get-started/virtual-machine-install/vmw-player.rst

```diff
@@ -114,13 +114,13 @@ Create and configure a new VM
 
    .. figure:: figures/vmw-player/vmw-player-03.png
       :scale: 100%
-      :alt: VMware Workstation 14 Player - Select guest operating system type
+      :alt: VMware Workstation Player - Select guest operating system type
 
-      Figure 3: VMware Workstation 14 Player - Select guest operating system
+      Figure 3: VMware Workstation Player - Select guest operating system
       type
 
 #. Set the :guilabel:`Version` setting to
-   :guilabel:`Other Linux 3.x or later kernel 64-bit`.
+   :guilabel:`Other Linux x.x or later kernel 64-bit`.
```

Comment:
```suggestion
   :guilabel:`Other Linux 4.x or later kernel 64-bit`.
```

x.x is too generic in this case becasue the drop down has similarly named options for 3x, 2x, etc so 4.x should be mentioned specifically.

x.x is too generic unfortunately, bec

Reply:
Changed back to 4.x.  @puneetse if you can update the image for this (vmw-player-03.png) that would be great.